source/PlayerInfo.h

```diff
@@ -140,7 +141,7 @@ class PlayerInfo {
 	void AddShip(const std::shared_ptr<Ship> &ship);
 	// Buy or sell a ship.
 	void BuyShip(const Ship *model, const std::string &name, bool isGift = false);
-	void SellShip(const Ship *selected);
+	void SellShip(const Ship *selected, bool isTaking = false);
```

Comment:
Does it make sense to make SellShip handle the monetary aspect and then call a method that just worries about removing the ship? Then we don't need to add a default parameter.

Reply:
Just noting that we don't want take ship to be paying anything when it does the work. Or at least, not obligatorily. At least sometimes this will be situations where it is inappropriate to have the player being monetarily compensated for three so being taken.

Reply:
yeah I wasn't going to add that, if a content creator wants to pay the player a given amount they wont just pay the value of the ship, they'll pay more probably so may as well let them choose any amount, and make this no actually give money to the player

Reply:
Yes, that is exactly my thought. We don't need extra complexity there.